lookout/style/format/analyzer.py

```diff
@@ -428,7 +428,15 @@ def _generate_token_fixes(
         for line_number, line in self._group_line_nodes(
                 y, y_pred, vnodes_y, new_vnodes, rule_winners):
             line_ys, line_ys_pred, line_vnodes_y, new_line_vnodes, line_winners = line
-            new_code_line = code_generator.generate_new_line(new_line_vnodes)
+            try:
+                new_code_line = code_generator.generate_new_line(new_line_vnodes)
+            except Exception as e:
```

Comment:
```suggestion
            except:
```

Reply:
No, Vadim, I should use `Exception` because you also have  `BaseException` as the highest level. I do not want to catch `SystemExit` `KeyboardInterupt` and `GeneratorExit`